addons/l10n_it_edi/i18n/it.po

```diff
@@ -141,31 +176,36 @@ msgstr "Conto bancario non trovato, informazioni utili dal file XML:"
 
 #. module: l10n_it_edi
 #: model_terms:ir.ui.view,arch_db:l10n_it_edi.res_config_settings_view_form
-msgid "By checking this box, I accept that Odoo may process my invoices."
+msgid "By checking this box, I accept that Odoo may process my e-invoices."
 msgstr "Selezionando questa casella, accetto che Odoo invii le mie fatture."
 
+#. module: l10n_it_edi
+#. odoo-python
+#: code:addons/l10n_it_edi/models/account_edi_format.py:0
+#, python-format
+msgid ""
+"Cannot apply Reverse Charge to a bill which contains both services and "
+"goods."
+msgstr "Impossibile applicare il Reverse Charge a una fattura che contiene"
+"sia servizi che merci."
+
 #. module: l10n_it_edi
 #: model:ir.model.fields,field_description:l10n_it_edi.field_res_company__l10n_it_codice_fiscale
 #: model:ir.model.fields,field_description:l10n_it_edi.field_res_partner__l10n_it_codice_fiscale
 #: model:ir.model.fields,field_description:l10n_it_edi.field_res_users__l10n_it_codice_fiscale
-msgid "Codice Fiscale"
+msgid "Tax Code"
 msgstr "Codice Fiscale"
 
 #. module: l10n_it_edi
 #: model:ir.model.constraint,message:l10n_it_edi.constraint_res_partner_l10n_it_codice_fiscale
-msgid "Codice fiscale must have between 11 and 16 characters."
+msgid "The tax code (Codice Fiscale) must have between 11 and 16 characters."
 msgstr "Il Codice Fiscale deve avere fra 11 e 16 caratteri."
 
 #. module: l10n_it_edi
 #: model:ir.model,name:l10n_it_edi.model_res_company
 msgid "Companies"
 msgstr "Aziende"
 
-#. module: l10n_it_edi
-#: model:ir.model.fields,field_description:l10n_it_edi.field_res_company__l10n_it_address_send_fatturapa
-msgid "Company PEC-mail"
-msgstr "Indirizzo PEC aziendale"
-
```

Comment:
PEC mail is no more. It's not exported by Odoo in the template file, so you won't see it here anymore.